build.gradle

```diff
@@ -30,6 +31,8 @@ configure( rootProject ) {
     apply from: "$rootDir/gradle/tools.gradle"
 }
 
+// TODO: Configure all lib projects here, applying "$rootDir/gradle/typescript.gradle", when moved to TS
```

Comment:
this an only be merged to master when all type definitions are ready.

Reply:
Should I remove this comment then? Maybe move it to an issue?

Reply:
Sure. I'm just want to make sure we won't forget about this after the epic is finished, because it's not probably me, who will add types to other libraries.